src/category_theory/limits/shapes/wide_pullbacks.lean

```diff
@@ -371,4 +371,82 @@ end
 
 end wide_pushout
 
+variable (J)
+
+/-- The obvious functor `wide_pullback_shape J ⥤ (wide_pushout_shape J)ᵒᵖ` -/
+@[simps]
+def wide_pullback_shape_op : wide_pullback_shape J ⥤ (wide_pushout_shape J)ᵒᵖ :=
+{ obj := λ X, op X,
+  map := λ X Y f, begin
+    apply quiver.hom.op,
```

Comment:
```suggestion
  map := λ X Y f,
  begin
    apply quiver.hom.op,
```

You could also consider moving the definition of `map` into a separate declaration so that you can use the equation compiler instead of having to make a definition in tactic mode:
```lean
def wide_pullback_shape_op_map : Π (X Y : wide_pullback_shape J),
  (X ⟶ Y) → ((op X : (wide_pushout_shape J)ᵒᵖ) ⟶ (op Y : (wide_pushout_shape J)ᵒᵖ))
| _ _ (wide_pullback_shape.hom.id X) := quiver.hom.op (wide_pushout_shape.hom.id _)
| _ _ (wide_pullback_shape.hom.term j) := quiver.hom.op (wide_pushout_shape.hom.init _)
```

Reply:
Ok.